src/main/java/com/google/sps/servlets/RecipeServlet.java

```diff
@@ -73,121 +81,129 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String data = request.getReader().lines().collect(Collectors.joining(System.lineSeparator()));
     Recipe newRecipe = gson.fromJson(data, Recipe.class);
-    if (newRecipe.content == null || newRecipe.title == null) {
+    newRecipe.metadata = gson.fromJson(data, RecipeMetadata.class);
```

Comment:
Is this line really necessary? Shouldn't everything be gotten by `Recipe newRecipe = gson.fromJson(data, Recipe.class)`?

Reply:
I couldn't get it to work

Reply:
Only the content field was populated, it didn't then go in and populate the Metadata members

Reply:
That's weird, I got it to work on my end. I set up the frontend request to have a body as follows:
```
{
  metadata: {title: [insert title here]},
  content: [insert content here]
}
```

Reply:
Huh it was probably a frontent problem then